### PR TITLE
Move translations from plone.app.locales to here

### LINKED ISSUE #11: Change translation domain to something else?
Now for Plone 6.1 we have removed Products.validation sources from buildout.coredev:

https://github.com/plone/buildout.coredev/commit/f7457fc0abdd01691b60cfbc489600c58d92c4ac

This means that the script that updates the translations from the sources will not take those translations (and it will delete the pre-existing ones).

If we change the [translation domain](https://github.com/plone/Products.validation/blob/master/Products/validation/i18n.py#L8) from `plone` to something else, we will need to bring the translated strings from `plone.app.locales` to here. And then we have another package where the translations should be added.

This also means that we need to create a separate branch in `plone.app.locales` for Plone 6.0.x and 6.1.x, which is not bad and perhaps we will end up needing that.

Or, we can rollback the removal of Products.validation in buildout.coredev.

Any comment @mauritsvanrees @jensens ?

### PR DESCRIPTION
**Pull request by erral**
Fixes #11

### CODE REVIEW

**Comment by erral** [comment]
@jenkins-plone-org please run jobs

File: Products/validation/locales/README.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Adding and updating locales
+---------------------------
+
+For every language you want to translate into you need a
+locales/[language]/LC_MESSAGES/collective.task.po
```
Comment (mauritsvanrees) [inline Products/validation/locales/README.rst]: I see some mentions of collective.task instead of plone.

Comment (erral) [reply Products/validation/locales/README.rst]: it was a copy paste... I'll fix it right away

**Comment by erral** [comment]
@jenkins-plone-org please run jobs

**Review by gforcada — APPROVED** [review]

**Comment by erral** [comment]
I am merging this to prepare the updated plone.app.locales

**Pull request merged**